packages/lib-utils/src/components/list-view/ListView.tsx

```diff
@@ -1,38 +1,26 @@
 import type { AnyObject } from '@monorepo/common';
-import {
-  Pagination,
-  PaginationVariant,
-  SearchInput,
-  Select,
-  SelectOption,
-  SelectVariant,
-  Toolbar,
-  ToolbarContent,
-  ToolbarItem,
-  ToolbarItemVariant,
-} from '@patternfly/react-core';
-import { FilterIcon } from '@patternfly/react-icons';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+import type {
+  ConditionalFilterItem,
+  FilterChipGroup,
+} from '@redhat-cloud-services/frontend-components';
```

Comment:
If we only intend to consume a couple of React components and types from ^^ library, wouldn't it make more sense to mark this library as non-external? This way, the code we import from it would become part of the generated SDK code (basically treating it as a dev dependency).